mapsforge-core/src/main/java/org/mapsforge/core/util/MercatorProjection.java

```diff
@@ -216,9 +210,10 @@ public static double latitudeToPixelY(double latitude, byte zoomLevel, int tileS
      * @return the pixel Y coordinate of the latitude value.
      */
     public static double latitudeToPixelY(double latitude, long mapSize) {
-        double sinLatitude = Math.sin(latitude * (Math.PI / 180));
+        double sinLatitude = Math.sin(Math.toRadians(latitude));
         // FIXME improve this formula so that it works correctly without the clipping
         double pixelY = (0.5 - Math.log((1 + sinLatitude) / (1 - sinLatitude)) / (4 * Math.PI)) * mapSize;
+//        double pixelY = (0.5 - Math.log(Math.tan(Math.toRadians(45 + latitude / 2))) / (2 * Math.PI)) * mapSize;
```

Comment:
@Sublimis What is this commented code?

Reply:
Some leftover, sorry, while trying to fix something. It is just another way of doing the exact same calculation in the line above.